test/system/200-pod.bats

```diff
@@ -339,6 +339,37 @@ EOF
     run_podman rmi $infra_image
 }
 
+@test "podman pod create should write its pod id to the file specified by --out" {
+    filename=$(mktemp -p ${BATS_TEST_TMPDIR} stdoutXXXXXXXX)
```

Comment:
No need for this, you can just use `outfile=${PODMAN_TMPDIR/out-results` or something like that.

Also, please use a more human-friendly variable name than `filename`

Reply:
got it. i'll squash all of these commits once all the reviews come in.

Reply:
I had to revert your suggestion within the "001-basic.bats" set of tests, as the `basic_setup()` function isn't called to assign PODMAN_TMPDIR. Is there a different way, or should I move the logic within `helpers.bash` (that assigns PODMAN_TMPDIR) somewhere so that it's scoped by `001-basic.bats`? Perhaps I should just duplicate basic_setup's TMPDIR code inside the `setup()` function for "001-basic.bats" despite my test being the only one using it?